examples/Makefile.rules

```diff
@@ -135,9 +135,14 @@ images: $(BINARY).images
 flash: $(BINARY).flash
 
 $(LDSCRIPT):
-    ifeq (,$(wildcard $(LDSCRIPT)))
-        $(error Unable to find specified linker script: $(LDSCRIPT))
-    endif
+	ifeq (,$(wildcard $@))
+		$(error Unable to find specified linker script: $(LDSCRIPT))
+	endif
+
+$(LIB_DIR)/lib$(LIBNAME).a:
+	ifeq (,$(wildcard $@))
+		$(error libopencm3 has not yet been compiled. Run `make` in $(OPENCM3_DIR))
```

Comment:
I'd actually rather it just ran make honestly...  instead of the error, do $(MAKE) -C $(OPENCM3_DIR)...  I actually don't really know why people need this in the examples/makefile.rules at all.  the top level makefile does the right thing, and if people are ignoring the top level makefile, ignoring the readme, and then complaining when they jump down a few levels, I'm not super sure I care much about their woes. 

Still, if its fixable, and I don't have to do it, sure why not?